README.md

```diff
@@ -6,93 +6,123 @@
-| selectedSections | | `[]` | An array of section IDs for the map to initially highlight by default when it is rendered. |
-| ticketGroups | | `[]` | An array of ticket groups to be used for section pricing. This API was designed for you to directly pass into the client library the response from the [Ticket Evolution `/v9/ticket_groups` endpoint](https://ticketevolution.atlassian.net/wiki/spaces/API/pages/9469962/Ticket+Groups+Index). Expects each ticket group to adhere to the [TicketGroup interface](#interfaces). |
+This is the same effect used for clicking on a section to select it. It colors the section and will not revert back by hovering off it or calling `unhighlightSection`. Calls the `onSelection` callback with the updated array of selected sections.
 
-# Public API
+#### `deselectSection(section: string)`
 
-| Name | Input | Output | Notes
-| - | - | - | - |
-| updateTicketGroups | `TicketGroup[]` | `void` | Changes the collection of ticket groups in the map used to calculate available sections and section prices. Useful if you have a feature for filtering ticket groups and you want the map to update.
-| highlightSection | `Section` | `void` | Temporarily colors the given section by making it more opaque. This is the same effect used for hovering on a section. |
-| unhighlightSection | `Section` | `void` | Removes the highlight effect of the given section, if it's not selected, by reverting it back to its base transparency. This is the same effect used for hovering off on a section. |
-| selectSection | `Section` | `void` | This is the same effect used for clicking on a section to select it. It colors the section and will not revert back by hovering off it or calling `unhighlightSection`. Calls the `onSelection` callback with the updated array of selected sections. |
-| deselectSection | `Section` | `void` | This is the same effect used for clicking on a section to deselect it. It reverts the color of the section and is the only way to unhighlight a selected section. Calls the `onSelection` callback with the updated array of selected sections. |
+This is the same effect used for clicking on a section to deselect it. It reverts the color of the section and is the only way to unhighlight a selected section. Calls the `onSelection` callback with the updated array of selected sections.
 
 # Interfaces
 
-| Name | Properties |
-| - | - |
+| Name        | Properties                                                                                                            |
+|-------------|-----------------------------------------------------------------------------------------------------------------------|
 | TicketGroup | `{`<br>&nbsp;&nbsp;&nbsp;&nbsp;`tevo_section_name: string;`<br>&nbsp;&nbsp;&nbsp;&nbsp;`retail_price: number;`<br>`}` |
-| Section | `string` |
+| Section     | `string`                                                                                                              |
```

Comment:
You can get rid of this row since you're typing the sections above directly to strings.